CMakeLists.txt

```diff
@@ -164,30 +164,50 @@ find_package(CURL REQUIRED)
 include_directories(${CURL_INCLUDE_DIRS})
 link_directories(${CURL_LIBRARY_DIRS})
 
-# set(Python_ADDITIONAL_VERSIONS 3.6 3.5 3.4)
-
-find_package(PythonInterp)
-find_package(PythonLibs)
+if (${CMAKE_VERSION} VERSION_LESS "3.13.3")
+    find_package(PythonInterp)
+    set(Python_Interpreter_FOUND ${PYTHONINTERP_FOUND})
+    set(Python_EXECUTABLE ${Python_EXECUTABLE})
```

Comment:
Assignment is not required in this case or you mean PYTHON_EXECUTABLE

Reply:
Thanks, it was supposed to be all uppercase.